src/Header.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const StyledNav = styled.nav`
+  border: 2px solid red;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 25px 80px 10px 80px;
```

Comment:
Why do you need top and bottom padding here?

Reply:
replaced padding-top with margin-top. without margin-top my nav is not aligning vertically.

Reply:
Ah ok, makes sense. An alternative would be to set a fixed height to the header. I guess it's 100px but you'd need to confirm with the designs. Since you already use `align-items: center` it would work. Otherwise, you'd need to adjust the margins every time you change the content (e.g. font-size of the links or height of the logo). Setting the height would be more future-proof